commit/merkleroot/types.go

```diff
@@ -132,6 +132,9 @@ type Outcome struct {
 	ReportTransmissionCheckAttempts uint                          `json:"reportTransmissionCheckAttempts"`
 	RMNReportSignatures             []cciptypes.RMNECDSASignature `json:"rmnReportSignatures"`
 	RMNRemoteCfg                    rmntypes.RemoteConfig         `json:"rmnRemoteCfg"`
+	//TODO: Calculate the bitmap
+	// This is a bitmap where ith bit represents how the v value should be for ith signature
+	RmnRawVs cciptypes.BigInt `json:"rmnRawVs"`
```

Comment:
nit, but all rmn related fields are capitalized (e.g. `RMNRemoteCfg`), let's keep it aligned

Reply:
Fixed